app/models/guest.rb

```diff
@@ -1,5 +1,5 @@
 class Guest < ActiveRecord::Base
-  validates :email, uniqueness: true
+  # validates :email, uniqueness: true
```

Comment:
This was raising errors because multiple guests could not have `nil` for their email address